e2e/testdata/fn-render/path-index-ancestor/.expected/config.yaml

```diff
@@ -12,8 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This package is nonIdempotent because functions in the subpackage (db)
-# do not have knowledge of the resources in the parent package. Therefore
-# when it runs twice, it tries to recreate the same resource in the parent,
-# resulting in a filepath conflict and throwing an error.
-nonIdempotent: true
\ No newline at end of file
+exitCode: 1
+stdErr: "Error: adjust relative path: cannot modify resources outside of package at path ../deployment_httpbin.yaml"
```

Comment:
"adjust relative path" seems out of palce

Reply:
"adjust relative path" no longer shows up in the error message